autonomy/cli/service.py

```diff
@@ -142,10 +142,10 @@ def _register(  # pylint: disable=too-many-arguments
 @hwi_flag
 @password_decorator
 @click.option(
-    "-d",
-    "--deployment-payload",
-    type=int,
-    help="Deployment payload value",
+    "-f",
+    "--fallback-handler",
+    type=str,
+    help="Fallback handler address for the gnosis safe multisig",
```

Comment:
Provide custom fallback handler using `-f/--fallback-handler` flag, eg

`autonomy deploy service ... --fallback-handler ADDRESS`

default is `0xf48f2b2d2a534e402487b3ee7c18c33aec0fe5e4`